src/led-element.ts

```diff
@@ -18,11 +18,17 @@ export class LEDElement extends LitElement {
   @property() color = 'red';
   @property() lightColor: string | null = null;
   @property() label = '';
+  @property() flipped = false;
```

Comment:
Let's specify that it's a boolean input, like here: https://github.com/wokwi/wokwi-elements/blob/9790133c7ac9a2f77eb66a2beb2aa26ceadd1937/src/7segment-element.ts#L21